pkg/repository/jsonb.go

```diff
@@ -1,16 +1,16 @@
 package repository
 
 import (
+	"bytes"
 	"fmt"
-	"strings"
 )
 
 func ValidateJSONB(jsonb []byte, fieldName string) error {
 	if len(jsonb) == 0 {
 		return nil
 	}
 
-	if strings.Contains(string(jsonb), "\u0000") {
+	if bytes.Contains(jsonb, []byte("\u0000")) {
```

Comment:
fyi @icbd this didn't work so I reverted it - not completely sure why though tbh, it seemed fine at first glance

Reply:
I think this broke a unit test btw, but I validated the behavior with an E2E test. Let me know if you disagree!